oneill-c/rock-paper-scissors
# Fixes issues for netlify deployment

**Depends on**

- [x] List of dependent unmerged PRs
      ...

**What's included**

- [x] List of what is included
      ...

**Test steps**

e.g.

1. Navigate to registration page
2. Enter name in fields
3. Click submit
4. Assert that success page is shown
5. Some really special case that you want to be checked
   ...
